assignment_1/basher666_print_squares/src/basher666_numbers.cpp

```diff
@@ -0,0 +1,28 @@
+#include "ros/ros.h"
+#include "std_msgs/Int64.h"
+#include <sstream>
+#include <bits/stdc++.h>
```

Comment:
You need not import everything. _Very few_ functions are required for this node, and it makes sense to import only those header files which are actually required.